pages/create-expense.js

```diff
@@ -0,0 +1,420 @@
+                                ...this.state.expense,
+                                tags: this.state.tags,
+                                createdByAccount: this.props.data.loggedInAccount,
+                              }}
+                            />
+                            {this.state.error && (
+                              <MessageBox type="error" withIcon mt={3}>
+                                {this.state.error.message}
+                              </MessageBox>
+                            )}
+                            <StyledButton
+                              buttonStyle="primary"
+                              mt={3}
+                              data-cy="submit-expense-btn"
+                              onClick={this.onSummarySubmit}
+                            >
+                              <FormattedMessage id="ExpenseForm.Submit" defaultMessage="Submit expense" />
+                            </StyledButton>
+                          </div>
+                        )}
+                      </Box>
+                    )}
+                  </Box>
+                  <Box mt={4} width={270}>
+                    <H5 mb={3}>
+                      <FormattedMessage id="CollectiveBalance" defaultMessage="Collective balance" />
+                    </H5>
+                    <Container borderLeft="1px solid" borderColor="green.600" pl={3} fontSize="H5" color="black.500">
+                      {data.loading ? (
+                        <LoadingPlaceholder height={28} width={75} />
+                      ) : (
+                        <FormattedMoneyAmount
+                          currency={collective.currency}
+                          amount={collective.balance}
+                          amountStyles={{ color: 'black.800' }}
+                        />
+                      )}
+                    </Container>
+                    <Box mt={50}>
+                      <H5 mb={3}>
+                        <FormattedMessage id="Tags" defaultMessage="Tags" />
+                      </H5>
+                      <StyledInputTags inputId="expense-tags" onChange={this.setTags} disabled />
+                    </Box>
+                    <ExpandableExpensePolicies host={host} collective={collective} mt={50} />
+                    <Box mt={50}>
+                      <CreateExpenseFAQ
+                        withBorderLeft
+                        withNewButtons
+                        titleProps={{ fontSize: 'H5', fontWeight: 500, mb: 3 }}
+                      />
+                    </Box>
+                  </Box>
+                </Flex>
+              </Box>
+              <MobileCollectiveInfoStickyBar isLoading={data.loading} collective={collective} host={host} />
+            </Container>
+          </React.Fragment>
+        </CollectiveThemeProvider>
+      </Page>
+    );
+  }
+}
+
+const getData = graphql(
+  gqlV2`
+    query CreateExpense($collectiveSlug: String!) {
```

Comment:
I'm not a big fan of having queries with action names, this would be OK for the mutation.
Maybe `CreateExpenseForm`?

Reply:
Yep correct, the main use we have for this name is to check things in the log. So it should make it clear that the intent is to fetch data for the create expense page.